tests/resources/session-tests-template/suite.xml

```diff
@@ -70,6 +70,21 @@
       <datum id="case_id" nodeset="instance('casedb')/casedb/case[@case_type='pregnancy'][@status='open']" value="./@case_id" detail-select="m0_case_short" detail-confirm="m0_case_long" autoselect="true"/>
     </session>
   </entry>
+    <entry>
+        <form>http://openrosa.org/formdesigner/1A0EDB0F-D139-47F2-B06C-FF1203DC5AC4</form>
+        <command id="m0-f3">
+            <text>
+                <locale id="forms.m0f3"/>
```

Comment:
Can we replace this with a non-locale'd text? (we don't use locale stuff in mockapps generally, for clarify)